src/main.cpp

```diff
@@ -3981,7 +3981,7 @@ std::string GetWarnings(const std::string& strFor)
     string strRPC;
 
     if (!CLIENT_VERSION_IS_RELEASE)
-        strStatusBar = _("This is a pre-release test build - use at your own risk - do not use for mining or merchant applications");
+        strStatusBar = "This is a pre-release test build - use at your own risk - do not use for mining or merchant applications";
```

Comment:
It seems this is not copy-paste able in the qt client. I imagine those would be hard to translate for some users.